portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,71 @@
 
 package com.google.sps.servlets;
 
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final String COMMENT_NAME = "name";
+  private static final String COMMENT_TEXT = "comment";
+  private static final String COMMENT_TIMESTAMP = "datePosted";
+
+  List<Comment> comments = new ArrayList<>();
```

Comment:
is this intended to be stored across all the requests? I'm not sure why this is here and not in a constructor, or init.

Additionally, if it is meant to be longer term stored, consider making this a private class member.

Reply:
I'm not sure either! I suppose I just completely forgot my Java syntax and best practices!